File: nutron-nutrition/src/main/java/com/lazysyntax/nutron/nutrition/model/dto/FoodDto.java
package com.lazysyntax.nutron.nutrition.model.dto;

import lombok.*;

import java.util.Objects;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class FoodDto {
    private String foodId;
    private String barcode;
    private String userId;
    private String name;
    private String nameEs;
    private String nameEn;
    private NutrimentsDto nutriments;
    private String nutriscoreGrade;
    private String brands;
}

File: nutron-app-kmp/composeApp/src/commonMain/kotlin/com/lazysyntax/nutron/data/remote/food/FoodDto.kt
package com.lazysyntax.nutron.data.remote.food

import com.lazysyntax.nutron.data.remote.meal.NutrimentsDto
import kotlinx.serialization.Serializable

@Serializable
data class FoodDto(
    val foodId: String? = null,
    val barcode: String?,
    val userId: String?,
    val name: String?,
    val nameEs: String?,
    val nameEn: String?,
    val nutriments: NutrimentsDto?,
    val nutriscoreGrade: String?,
    val brands: String?
)
